Bottom: 99c37f508e9c7909657ff976a6cb5906008799e3
Top:    4e4407579988f198818c5e258fdbf0595c2fc5a0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 20:10:09 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index a9a537e..3893e56 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
